blake2/src/consts.rs

```diff
@@ -13,6 +13,7 @@ pub static SIGMA : [[usize; 16]; 12] = [
     [ 14, 10,  4,  8,  9, 15, 13,  6,  1, 12,  0,  2, 11,  7,  5,  3 ],
 ];
 
+#[cfg_attr(feature = "cargo-clippy", allow(unreadable_literal))]
```

Comment:
Can't we use `#![cfg_attr(..))]` on the whole file? I think it will be less repetitive. Same for other `consts.rs` files.